lib/src/marker_cluster_layer.dart

```diff
@@ -50,17 +50,17 @@ class _MarkerClusterLayerState extends State<MarkerClusterLayer>
         widget.map.getPixelOrigin();
   }
 
-  Point _getPixelFromMarker(MarkerNode marker, [LatLng customPoint]) {
+  Point _getPixelFromMarker(MarkerNode marker, [LatLng? customPoint]) {
     final pos = _getPixelFromPoint(customPoint ?? marker.point);
     return _removeAnchor(pos, marker.width, marker.height, marker.anchor);
   }
 
-  Point _getPixelFromCluster(MarkerClusterNode cluster, [LatLng customPoint]) {
+  Point _getPixelFromCluster(MarkerClusterNode cluster, [LatLng? customPoint]) {
     final pos = _getPixelFromPoint(customPoint ?? cluster.point);
 
     Size size = getClusterSize(cluster);
     Anchor anchor =
-        Anchor.forPos(widget.options.anchor, size.width, size.height);
+        Anchor.forPos(widget.options.anchor!, size.width, size.height);
```

Comment:
I think there should be no `!` on `widget.options.anchor!`. It's erroring for me locally and the forPos method handles null.

Reply:
removed